samples/client/petstore/groovy/bin/main/org/openapitools/api/ApiUtils.groovy

```diff
@@ -0,0 +1,54 @@
+package org.openapitools.api
```

Comment:
I noticed some changes (newly-added files) for Groovy Petstore client. Which script did you run to update the Groovy Petstore client?

I run `./bin/groovy-petstore.sh` with the latest master but don't see any changes.

Please undo these changes if not needed.